orttraining/orttraining/test/python/orttraining_test_orttrainer_frontend.py

```diff
@@ -404,3 +443,93 @@ def testLRSchedulerUpdateImpl(lr_scheduler, expected_values):
         assert len(lr_list) == 1
         assert_allclose(lr_list[0],
                         expected_values[step], rtol=rtol, err_msg="lr mismatch")
+
+def my_loss(x, target):
```

Comment:
my_loss, transformer_model_description, batchify, get_batch should be in a separate file in  /examples/python/pytorch_transformer